main.go

```diff
@@ -707,6 +708,7 @@ func mustPrepareService(generateAutoKeyFile bool) (*service.Service, service.Boo
 		ArangodPath:             arangodPath,
 		ArangoSyncPath:          arangoSyncPath,
 		ArangodJSPath:           arangodJSPath,
+		AdvertisedEndpoint:      advertisedEndpoint,
```

Comment:
Secondly I suggest to allow the use of "http" & "https" schemes in here and automatically transform them to "tcp" & "ssl"

Reply:
I strongly disagree on this. The starter is supposed to make life easier and having think about "tcp://" or "ssl://" where everyone knows about "http://" and "https://" does not help.
Also we don't have to check anything except replace an "http://" prefix with "tcp://" (and https...)